### DISCUSSION TITLE
Change colour of favourites

### DISCUSSION THREAD

**Discussion by c7nb4e** [osmandapp/OsmAnd#17003] [Q&A]
Hi

I can't seem to change the colour of favourites that I have copied from an imported track. In the folder Import, I tap on my track, then I tap on Waypoints. Next, I tap on the three dots next to the folder named waypoints. In the dialog that appears now, I tap on change colour and select my desired one, e.g. grey. I can now see the colour of the waypoints change on the map. Now I copy the waypoints to the favourites, but now the copied waypoints have a totally different colour, red.

If I now tap on the pencil and then select my newly created folder, and try to change the colour here, nothing happens.

I'm using iOS. Does anyone have a solution for this?

Thanks, Chris

**Comment by dmpr0** [comment]
Hello!
Currently OsmAnd Maps on iOS doesn't have option to change appearance of whole group of favorite points. I can reproduce the issues with changed color after copying waypoints to favorites, but after that I can change the color of individual favorites.

**Reply by dmpr0** [reply]
I added the issues about copying points https://github.com/osmandapp/OsmAnd-iOS/issues/2813
